include/ddc/kernels/splines/greville_interpolation_points.hpp

```diff
@@ -20,8 +20,8 @@ namespace ddc {
  * A class which provides helper functions to initialise the Greville points from a B-Spline definition.
  *
  * @tparam BSplines The bspline class relative to which the Greville points will be calculated.
- * @tparam BcXmin The (left) boundary condition that will be used to build the splines.
- * @tparam BcXmax The (right) boundary condition that will be used to build the splines.
+ * @tparam BcXmin The lower boundary condition that will be used to build the splines.
```

Comment:
Shall we also rename `BcXmin` ? `BcLower` ?

Reply:
I agree. What about rmin(), rmax(), mixed_derivs_min1_max2, s_nbc_xmin ? Maybe in another MR just in case of problem with one of those commits ?

Reply:
Mmh, that would feel weird to me to go for `rupper`/`rlower`. Maybe we should do the reverse then, change the documentation to match the current notation `xmin`/`xmax` ?

Reply:
Or just use Lower/Upper in types (including BcLower, BcUpper) and min/max for variable ?

Reply:
So just `BcXmin` and `BcXmax` ? I am ok with it